### PR TITLE
Update pgbackrest.md, see DISTPG-749

### PR DESCRIPTION
**Pull request by laudares**
Implementing the changes described in https://perconadev.atlassian.net/browse/DISTPG-749 and adding general clarity to the procedure.

### CODE REVIEW

File: docs/solutions/pgbackrest.md [hunk 1]
```diff
@@ -268,42 +426,84 @@ Run the following command on `node1`, `node2` and `node3`.
...
-        recovery_target_timeline: latest
-        restore_command: pgbackrest --config=/etc/pgbackrest.conf --stanza=cluster_1 archive-get %f "%p"
-      use_pg_rewind: true
-      use_slots: true
-    retry_timeout: 10
-    slots:
-      percona_cluster_1:
-        type: physical
-    ttl: 30
+    The pgBackRest daemon listens on port `8432` by default:
```
Comment (nastena1606) [inline docs/solutions/pgbackrest.md:435]:
```suggestion
    The pgBackRest daemon listens on port `8432` by default:
    
```

File: docs/solutions/pgbackrest.md [hunk 2]
```diff
@@ -268,42 +426,84 @@ Run the following command on `node1`, `node2` and `node3`.
...
+          use_pg_rewind: true
+          use_slots: true
+        retry_timeout: 10
+        slots:
+          percona_cluster_1:
+            type: physical
+        ttl: 30
+        ```
+
+    *NOTE*: When configuring a PostgreSQL server that is not managed by Patroni to archive/restore WALs from a pgBackRest server, edit it's main configuration file directly and adjust the `archive_command` and `restore_command` variables as shown above.
```
Comment (nastena1606) [inline docs/solutions/pgbackrest.md]:
```suggestion
    *NOTE*: When configuring a PostgreSQL server that is not managed by Patroni to archive/restore WALs from a pgBackRest server, edit its main configuration file directly and adjust the `archive_command` and `restore_command` variables as shown above.
```

**Comment by laudares** [comment]
Guys, there's finally a few more things for me to fix in that doc...

**Review by laudares — COMMENTED** [review]
This procedure is ready to be published now

**Pull request merged**